src/sentry/integrations/slack/message_builder/help.py

```diff
@@ -9,18 +9,26 @@
 HEADER_MESSAGE = "Here are the commands you can use. Commands not working? Re-install the app!"
 DM_COMMAND_HEADER = "*Direct Message Commands:*"
 CHANNEL_COMMANDS_HEADER = "*Channel Commands:*"
+HELP_COMMANDS_HEADER = "*Help Commands:*"
+HELP_COMMANDS_HEADER_MESSAGE = (
+    "These commands can be run in a direct message with the Sentry app or in a channel."
+)
 CONTACT_HEADER = "*Contact:*"
 GENERAL_MESSAGE = "Just want to learn more about Sentry? Check out our <https://docs.sentry.io/product/integrations/notification-incidents/slack/|documentation>."
 
 DM_COMMANDS = {
     "link": "Link your Slack identity to your Sentry account to receive notifications. You'll also be able to perform actions in Sentry through Slack.",
     "unlink": "Unlink your Slack identity from your Sentry account.",
-    "help": "View this list of commands.",
 }
 CHANNEL_COMMANDS = {
     "link team": "Get your Sentry team's issue alert notifications in the channel this command is typed in.",
     "unlink team": "Unlink a team from the channel this command is typed in.",
```

Comment:
```suggestion
    "link team": "Get your Sentry team's issue alert notifications in this channel.",
    "unlink team": "Unlink a team from this channel.",
```
I think it's obvious it's in the channel the command is typed in, so we can just say "this channel"